### DISCUSSION TITLE
Trying to run novadrop-run client but game crashes

### DISCUSSION THREAD

**Discussion by wanqwer** [vezel-dev/novadrop#79] [Questions] [answered]
I'm trying to start the game using novadrop-run client with parameters but 75% is the furthest I got so far.
https://prnt.sc/A_orGqMvfd0w

At this point console does not show errors so it is hard to debug what I did wrong. 
Game starts, goes past splash, shows gameforge animation, shows first Tera screen and then turns off before showing character list.

All parameters look OK to me. Any ideas what can be done here?
P.s. server is running fine, I can login and play with other launchers etc...

**Comment by alexrp** [comment, marked as answer]
I only ever tested this command with patch 115. I wouldn't be surprised if it doesn't work with other versions. It also wouldn't be too surprising if it regressed since there was no way for me to test it after official servers went offline, since nobody runs patch 115 servers.

You'd have to do some investigation to find out what's going wrong. I'd suggest adding logging in the various message methods in `ClientProcess`/`LauncherProcess`.
